sts-mock/src/token/validateServiceTokenRequest/validateServiceTokenRequest.ts

```diff
@@ -0,0 +1,40 @@
+import { errorResult, Result, successResult } from "../../utils/result";
```

Comment:
I think we need to have a conversation about how much request validation we want to do/is necessary in this mock. This can be enhanced in a separate PR.

Reply:
Discussed below.